packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -143,6 +143,13 @@ export default function EditPostPreferencesModal() {
 								) }
 								label={ __( 'Use theme styles' ) }
 							/>
+							<EnableFeature
+								featureName="enableChoosePatternModal"
+								help={ __(
+									'Enables the choose pattern modal when adding a new page.'
+								) }
+								label={ __( 'Enable choose pattern modal' ) }
+							/>
```

Comment:
I think the copy could be tweaked as a lot of users won't know what a modal is.

Something like "Show starter patterns" / "Shows starter patterns when creating a new page.".

Reply:
+1